doc/xsl/docbook-html-utf8.xsl.in

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0'?>
+<xsl:stylesheet version="1.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:import href="@XSLBASE@/html/docbook.xsl"/>
+<xsl:output method="html" encoding="UTF-8" indent="no"/>
```

Comment:
Since we're at it, I'd indent :)

Reply:
Or honour user variable, like the chunked-html xsl does. We recently switcehd to indentend for chunked-html, using chunker.output.indent variable.

Reply:
speaking of which, if we use variables might as well use variable for output encoding, so we have chunked and single-page both capable of using a variable. And if we had to use the same variable name we might even use a common set of switches